openai-java-example/src/main/java/com/openai/example/AzureEntraIdExample.java

```diff
@@ -12,16 +12,22 @@ public final class AzureEntraIdExample {
     private AzureEntraIdExample() {}
 
     public static void main(String[] args) {
-        OpenAIClient client = OpenAIOkHttpClient.builder()
-                // Gets the API key from the `AZURE_OPENAI_KEY` environment variable
-                .fromEnv()
+        OpenAIOkHttpClient.Builder clientBuilder = OpenAIOkHttpClient.builder();
+
+        /* Azure-specific code starts here */
+        // You can either set 'endpoint' directly in the builder.
+        // or set the env var "AZURE_OPENAI_ENDPOINT" and use fromEnv() method instead
+        clientBuilder
+                .baseUrl(System.getenv("AZURE_OPENAI_ENDPOINT"))
                 // Set the Azure Entra ID
                 .credential(BearerTokenCredential.create(AuthenticationUtil.getBearerTokenSupplier(
-                        new DefaultAzureCredentialBuilder().build(), "https://cognitiveservices.azure.com/.default")))
-                .build();
+                        new DefaultAzureCredentialBuilder().build(), "https://cognitiveservices.azure.com/.default")));
+        /* Azure-specific code ends here */
+
+        OpenAIClient client = clientBuilder.build();
```

Comment:
Can we leave this code as it was? I think it's nicer to show users the "concise" way to build their client